src/components/Organisms/header/Header.js

```diff
@@ -3,30 +3,38 @@ import Icon from '../../atoms/Icon/Icon';
 import './Header.scss';
 import {Link} from 'react-router-dom';
 import {useTranslation} from 'react-i18next';
+import { Cart } from '../../Pages/Cart/Cart';
+import { CartHeader } from '../../molecules/HeaderCard/CartHeader';
 export default function Header(){
     // Used for text translation
-    const {t, i18n} = useTranslation();
+    const {t} = useTranslation();
+    //Function to update selected as per clicked element
+    const updateActiveStatus = (e) =>{
+        document.querySelector('.selected') && 
+        document.querySelector('.selected').classList.remove('selected');
+        (e.target.tagName === "A") ? 
+        e.target.classList.add('selected') : 
+        (e.target.tagName === "IMG" && document.querySelector('#header__nav-home-item_home-link').classList.add('selected'));
+    };
```

Comment:
This is anti-pattern stuff in terms of React. You manipulating DOM directly.

Reply:
Handled it through state and url updates now.